lib/screens/login_page.dart

```diff
@@ -0,0 +1,114 @@
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 25),
+              child: Container(
+                padding: const EdgeInsets.only(left: 10),
+                decoration: BoxDecoration(
+                  color: const Color.fromARGB(255, 198, 205, 205),
+                  border: Border.all(color: Colors.white),
+                  borderRadius: BorderRadius.circular(12),
+                ),
+                child: const TextField(
+                  decoration: InputDecoration(
+                      border: InputBorder.none,
+                      hintText: 'Email / Phone number'),
+                ),
+              ),
+            ),
+            const SizedBox(height: 10),
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 25),
+              child: Container(
+                padding: const EdgeInsets.only(left: 10),
+                decoration: BoxDecoration(
+                  color: const Color.fromARGB(255, 198, 205, 205),
+                  border: Border.all(color: Colors.white),
+                  borderRadius: BorderRadius.circular(12),
+                ),
+                child: const TextField(
+                  obscureText: true,
+                  decoration: InputDecoration(
+                      border: InputBorder.none, hintText: 'Password'),
+                ),
+              ),
+            ),
+            const SizedBox(height: 20),
+            Padding(
+              padding: const EdgeInsets.symmetric(horizontal: 25),
+              child: Container(
+                padding: const EdgeInsets.all(28),
+                decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(12),
+                ),
+                child: ElevatedButton(
+                    onPressed: () {
+                      Navigator.of(context).pushNamed(HomeScreen.route);
+                    },
+                    child: Text(
+                      "Sign In",
+                      style: GoogleFonts.exo2(
+                        fontSize: 18,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.black87,
+                      ),
+                    )),
+              ),
+            ),
+            const SizedBox(height: 10),
+            Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Text(
+                  "Forgot Password?",
+                  style: GoogleFonts.exo2(
+                      color: const Color.fromARGB(255, 20, 165, 217),
+                      fontWeight: FontWeight.bold),
+                ),
+                const SizedBox(height: 80),
+                const Image(
+                    image: AssetImage(
+                        "assets/example/images/icons8-snowflake-48.png")),
+              ],
+            )
+          ])),
+        ));
+  }
+
+  void navigateNextPage(BuildContext context) {}
```

Comment:
You could complete it like this:
```suggestion
  void navigateNextPage() {
    Navigator.of(context).pushNamed(HomeScreen.route);
  }
```

Reply:
this function (if connected to a real database) will also validate username and password for input and make sure it is correct before moving on, but for this project it is fine

Reply:
also to access context with this function, move it inside the build function